src/lib/coreboot_table.c

```diff
@@ -248,7 +248,9 @@ static void add_cbmem_pointers(struct lb_header *header)
 		{CBMEM_ID_ACPI_GNVS, LB_TAG_ACPI_GNVS},
 		{CBMEM_ID_VPD, LB_TAG_VPD},
 		{CBMEM_ID_WIFI_CALIBRATION, LB_TAG_WIFI_CALIBRATION},
+#if !CONFIG(TPM_MEASURED_BOOT_SPEC_LOG)
```

Comment:
Is this necessary? Code simply won't find this section if it isn't present. Talos2 doesn't have most of the rest anyway.

Reply:
It is present, but not in the format `coreboot` normally uses. I just didn't want to export table in incompatible format.

Reply:
Good point. In that case maybe a new ID should be added?

Reply:
Yeah, that's a better way of doing it.